.github/scripts/install-openssl.sh

```diff
@@ -6,12 +6,31 @@ os_name="$1"
 
 case "$os_name" in
 "Windows")
-  echo "Downloading OpenSSL installer..."
-  curl -L -o "openssl-installer.exe" "https://slproweb.com/download/Win64OpenSSL-3_4_2.exe"
-  echo "Installing OpenSSL..."
-  cmd.exe /c "openssl-installer.exe /verysilent /sp- /suppressmsgboxes /norestart /DIR=C:\OpenSSL"
-  export OPENSSL_LIB_DIR="C:\OpenSSL\lib\VC\x64\MT"
-  export OPENSSL_INCLUDE_DIR="C:\OpenSSL\include"
+  # initialize vcpkg.json. the builtin-baseline is the commit that contains required version.
+  # the commit hash can be found in https://github.com/microsoft/vcpkg with `git log --pretty=format:"%H %s" | grep openssl`
+  # can be verified with:
+  # web:
+  #     https://github.com/microsoft/vcpkg/blob/__COMMIT_HASH__/versions/o-/openssl.json
+  # command:
+  #     git show __COMMIT_HASH__:versions/o-/openssl.json
+  cat > vcpkg.json <<EOL
+{
+  "name": "agave",
+  "version-string": "0.1.0",
```

Comment:
Shouldn't this match the agave version?

Reply:
oh... thank you for bumping this up. maybe I can try to remove these lines. let me check

Reply:
good catch! removed useless lines: [7b049b9](https://github.com/anza-xyz/agave/pull/6824/commits/7b049b971e5afbb5fb1abc322cc43dcc2d483b21)